update.sh

```diff
@@ -1,7 +1,10 @@
 # Check for any updates of Linux
 sudo apt-get update
-# Update WhiteCore-Sim
+# go to /
 cd /
+# Run backup of current Working WC System
+tar czf backup.$(date +%Y%m%d-%H%M%S).tar.gz Github
```

Comment:
The concern here I believe is that this backs up the entire github directory.  It really should just back up the WhiteCore-Dev directory as the user might end up using the Github directory for other repositories to.